releases/release-1.25/release-notes/maps/pr-110948-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 110948
+releasenote:
+  text: JobTrackingWithFinalizers enabled by default. This feature allows to keep
```

Comment:
```suggestion
  text: `JobTrackingWithFinalizers` enabled by default. This feature allows to keep
```

Reply:
We can't use the backtick `  as the first character in a scalar, @ and ` are reserved characters https://yaml.org/spec/1.2.2/#rule-c-reserved

@Divya063 recently hit this issue here https://github.com/kubernetes/sig-release/pull/1975/files